flutter/lib/src/integrations/native_app_start_handler.dart

```diff
@@ -48,11 +51,23 @@ class NativeAppStartHandler {
       scope.span ??= rootScreenTransaction;
     });
 
+    /// Workaround to get TTFD when users called it before the root transaction is even created.
+    Future.delayed(const Duration(milliseconds: 1), () {
+      final reportedRootTTFDEndTimestamp =
+          options.timeToDisplayTracker.rootTransactionEndTimestamp;
+      if (reportedRootTTFDEndTimestamp != null) {
+        options.timeToDisplayTracker.reportFullyDisplayed(
+          spanId: rootScreenTransaction.context.spanId,
+          endTimestamp: reportedRootTTFDEndTimestamp.isAfter(appStartInfo.end)
+              ? reportedRootTTFDEndTimestamp
+              : appStartInfo.end,
+        );
+      }
+    });
```

Comment:
why do we need the 1ms delay?

Reply:
Can't you use a `Future.microtask` here? It's guaranteed to be run in order, but you don't need the delay.

Reply:
`Future.microtask` did not work, but i made this even simpler and more robust. If we know the ttfd at this point, we just inject it into the track method.